unime/src/routes/badges/[id]/+page.svelte

```diff
@@ -178,19 +164,19 @@
         </p>
       </div>
 
-      <!-- Metadata (Table: Credential Subject) -->
+      <!-- Contents (Table: Credential Subject) -->
```

Comment:
Did we change "metadata" to "contents" for a specific reason (just curious)?

Reply:
Yes, it was decided in a feedback session that "Contents" makes more sense than just the generic "Metadata".